src/components/tag-list/tag-list.jsx

```diff
@@ -0,0 +1,67 @@
+import React, {
+  useState,
+} from 'react';
+import PropTypes from 'prop-types';
+import styles from './tag-list.css';
+import { useRef } from 'react';
+import { useEffect } from 'react';
+
+export default function TagList(props) {
+  const [active, setActive] = useState(false);
+  const [focusIndex, setFocusIndex] = useState(0);
+
+  useEffect(() => {
+    if (active) {
+      props.refs.forEach((ref, index) => {
+        ref.current && ref.current.setIsParentActive(true);
```

Comment:
Since a `Tag` now has a `ref API`, we want to document how `TagList` uses the `ref` API.

Reply:
So maybe we only document that we want our child regs to implement the "parent active" and "self active" interface. This is gonna be reused in the future for all complex widgets but we don't need to get too ahead of ourselves with abstractions.